### DISCUSSION TITLE
Linux host assumes WSL

### DISCUSSION THREAD

**Discussion by mfgering** [doublecmd/doublecmd#805] [General]
I am trying to understand why dc on my ubuntu 22.04 host fails to open common files, e.g. foo.txt.
When I use the "Open Ctrl+Alt+Enter" context menu item on a file, I see these debug logs:
`
grep: /proc/sys/fs/binfmt_misc/WSLInterop: No such file or directory
WSL Interopability is disabled. Please enable it before using WSL.
grep: /proc/sys/fs/binfmt_misc/WSLInterop: No such file or directory
[error] WSL Interoperability is disabled. Please enable it before using WSL.
/usr/bin/wslview: line 216: /mnt/c/Windows/System32/reg.exe: No such file or directory
/usr/bin/wslview: line 308: [: -ge: unary operator expected
[error] This protocol is not supported before version 1903.

Why is it assuming it is running in a WSL task on Windows?

**Comment by alexx2000** [comment]
https://askubuntu.com/questions/1318261/nautilus-files-does-not-open-any-of-my-files
So the solution : `sudo apt remove wslu`

**Reply by mfgering** [reply]
Perfect! Thank you! 
For others: `sudo apt remove wslu` is the solution.
